src/deprecated/font-awesome-version.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * FontAwesome icons v5.15.4 is used in v3.12.6 and below.
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( ! function_exists( 'stackable_font_awesome_icons_version_set_default' ) ) {
+
+	/**
+	 * When upgrading to v3.12.7 and above, use FontAwesome icons v5.15.4 by default.
+	 * If new installation, use FontAwesome icons v6.4.2.
+	 *
+	 */
+	function stackable_font_awesome_icons_version_set_default( $old_version, $new_version ) {
+		if ( ! empty( $old_version ) && version_compare( $old_version, "3.12.6", "<" ) ) {
+			// Use FontAwesome icons v5.15.4 for users upgrading from v3.12.6 and below.
+			if ( ! get_option( 'stackable_fa_icons_version' ) || get_option( 'stackable_fa_icons_version' ) === '5.15.4' ) {
+				update_option( 'stackable_fa_icons_version', '5.15.4' );
+			}
+		} else if ( empty( $old_version ) ) {
+			// Use FontAwesome icons v6.4.2 for new installations.
+			update_option( 'stackable_fa_icons_version', '6.4.2' );
```

Comment:
I would recommend removing having to save the icon version for new installs. It would be better for new installs to have `stackable_fa_icons_version` blank (which translates to `false`), so that it means that if `stackable_fa_icons_version` is false, just use 6.4.2 
```suggestion
```